lexilla/include/SciLexer.h

```diff
@@ -167,102 +165,101 @@
 #define SCE_UNIVERSAL_FOUND_STYLE_EXT4 22
 #define SCE_UNIVERSAL_FOUND_STYLE_EXT5 21
 
-#define SCE_UDL_VERSION_MAJOR       2
-#define SCE_UDL_VERSION_MINOR       1
-#define SCE_UDL_VERSION_BUILD       0
-#define SCE_UDL_VERSION_REVISION    12
-
-#define SCE_USER_KWLIST_COMMENTS                    0
-#define SCE_USER_KWLIST_NUMBER_PREFIX1              1
-#define SCE_USER_KWLIST_NUMBER_PREFIX2              2
-#define SCE_USER_KWLIST_NUMBER_EXTRAS1              3
-#define SCE_USER_KWLIST_NUMBER_EXTRAS2              4
-#define SCE_USER_KWLIST_NUMBER_SUFFIX1              5
-#define SCE_USER_KWLIST_NUMBER_SUFFIX2              6
-#define SCE_USER_KWLIST_NUMBER_RANGE                7
-#define SCE_USER_KWLIST_OPERATORS1                  8
-#define SCE_USER_KWLIST_OPERATORS2                  9
-#define SCE_USER_KWLIST_FOLDERS_IN_CODE1_OPEN       10
-#define SCE_USER_KWLIST_FOLDERS_IN_CODE1_MIDDLE     11
-#define SCE_USER_KWLIST_FOLDERS_IN_CODE1_CLOSE      12
-#define SCE_USER_KWLIST_FOLDERS_IN_CODE2_OPEN       13
-#define SCE_USER_KWLIST_FOLDERS_IN_CODE2_MIDDLE     14
-#define SCE_USER_KWLIST_FOLDERS_IN_CODE2_CLOSE      15
-#define SCE_USER_KWLIST_FOLDERS_IN_COMMENT_OPEN     16
-#define SCE_USER_KWLIST_FOLDERS_IN_COMMENT_MIDDLE   17
-#define SCE_USER_KWLIST_FOLDERS_IN_COMMENT_CLOSE    18
-#define SCE_USER_KWLIST_KEYWORDS1                   19
-#define SCE_USER_KWLIST_KEYWORDS2                   20
-#define SCE_USER_KWLIST_KEYWORDS3                   21
-#define SCE_USER_KWLIST_KEYWORDS4                   22
-#define SCE_USER_KWLIST_KEYWORDS5                   23
-#define SCE_USER_KWLIST_KEYWORDS6                   24
-#define SCE_USER_KWLIST_KEYWORDS7                   25
-#define SCE_USER_KWLIST_KEYWORDS8                   26
-#define SCE_USER_KWLIST_DELIMITERS                  27
-#define SCE_USER_KWLIST_TOTAL                       28  // must always be <= KEYWORDSET_MAX
+#define SCE_UDL_VERSION_MAJOR 2
+#define SCE_UDL_VERSION_MINOR 1
+#define SCE_UDL_VERSION_BUILD 0
+#define SCE_UDL_VERSION_REVISION 12
 
-#define SCE_USER_TOTAL_DELIMITERS           8
-#define SCE_USER_TOTAL_KEYWORD_GROUPS       8
-#define SCE_USER_TOTAL_FOLDERS              6
+#define SCE_USER_KWLIST_COMMENTS 0
```

Comment:
This file is auto-generated by a Python script: https://github.com/ScintillaOrg/lexilla/blob/fa7721140059a3e16630893e3f5a6721f13063f7/scripts/LexFacer.py#L26

You should completely drop these changes if you decided to pitch this PR upstream: https://github.com/ScintillaOrg/lexilla